css/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+  /* background: #a5a5a5;
+    color: #fff; */
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.container {
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+h1 {
+  text-align: center;
+}
+
+#description {
+  text-align: center;
+  font-size: 20px;
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 10px;
+}
+
+.form-container {
+  display: flex;
+  justify-content: center;
+}
+
+form {
+  box-sizing: border-box;
+  padding: 20px;
+  background-color: #d1d1d1;
+  border-radius: 10px;
+  box-shadow: 4px 5px 10px gray;
+  overflow: hidden;
+}
+
+label {
+  font-size: 18px;
+  font-weight: bold;
+}
```

Comment:
[] kindly use the class selector because it can not be affected by the other selector or be overwritten